src/xfile.cc

```diff
@@ -532,7 +532,9 @@ bool xbaseOpen(const char* path)
 
     char workingDirectory[COMPAT_MAX_PATH];
     if (getcwd(workingDirectory, COMPAT_MAX_PATH) == nullptr) {
-        // FIXME: Leaking xbase and path.
+        // FIXME: Leaking xbase and path. Fixed?
```

Comment:
yeah, looks fixed.  we can remove the comment.  (normally you'd have a central cleanup place and use `goto`, but no need to get into that in this PR)